test/unit/axis-aligned-bounding-box.js

```diff
@@ -66,4 +66,209 @@ describe('AxisAlignedBoundingBoxCollider', function() {
     expect(displacement.x).to.be.closeTo(0, MARGIN_OF_ERROR);
     expect(displacement.y).to.be.closeTo(-1, MARGIN_OF_ERROR);
   });
+
+  describe('updateFromSprite()', function() {
+    var pInst, testAnimation;
+
+    beforeEach(function() {
+      pInst = new p5(function() {});
+      testAnimation = createTestAnimation(3);
+    });
+
+    afterEach(function() {
+      pInst.remove();
+    });
+
+    it('adopts dimensions from animationless sprite when no dimensions are given, scale 1', function() {
+      var sprite = pInst.createSprite(0, 0, 200, 210);
+      sprite.setCollider('aabb');
+      expect(sprite.collider.width).to.equal(200);
+      expect(sprite.collider.height).to.equal(210);
+      expect(sprite.collider.getBoundingBox().width).to.equal(200);
+      expect(sprite.collider.getBoundingBox().height).to.equal(210);
```

Comment:
Note: For the `AxisAlignedBoundingBox` and `OrientedBoundingBox` tests we expect the collider `width` and `height` to be unscaled values, and `getBoundingBox().width` and `getBoundingBox().height` to be scaled values.